package.json

```
@@ -7,12 +7,14 @@
   },
   "description": "VS Code support for the Simple API modeling toolkit",
```

I would personally write everything or nothign capitalzied
```suggestion
  "description": "VS Code support for the Simple API Modeling Toolkit",
```